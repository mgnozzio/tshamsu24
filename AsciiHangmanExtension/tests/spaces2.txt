hang man
h
a
n
g
 
m
a
n
