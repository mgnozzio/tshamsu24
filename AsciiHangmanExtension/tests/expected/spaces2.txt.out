Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ _ _ 
Please guess a letter: H
Puzzle to solve: H _ _ _ _ _ _ _ 
Please guess a letter: A
Puzzle to solve: H A _ _ _ _ A _ 
Please guess a letter: N
Puzzle to solve: H A N _ _ _ A N 
Please guess a letter: G
Puzzle to solve: H A N G _ _ A N 
Please guess a letter:  
Puzzle to solve: H A N G   _ A N 
Please guess a letter: M
Success!  Player 2 wins!
